Description: Start heartbeat after drbd
Author: Valentin Vidic <Valentin.Vidic@CARNet.hr>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=537982
Last-Update: 2017-10-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/heartbeat/init.d/heartbeat.in
+++ b/heartbeat/init.d/heartbeat.in
@@ -37,9 +37,9 @@
 # Short-Description: High-availability services.
 # Provides:		heartbeat HA
 # Required-Start:	$remote_fs $network $time $syslog
-# Should-Start:		openhpid
+# Should-Start:		openhpid drbd
 # Required-Stop:	$remote_fs $network $time $syslog
-# Should-Stop:		openhpid
+# Should-Stop:		openhpid drbd
 # Default-Start:	2 3 4 5
 # Default-Stop:		0 1 6
 ### END INIT INFO
--- a/heartbeat/init.d/heartbeat.service.in
+++ b/heartbeat/init.d/heartbeat.service.in
@@ -8,6 +8,8 @@
 Requires=basic.target
 Requires=network.target
 
+After=drbd.service
+
 [Install]
 WantedBy=multi-user.target
 
@@ -16,7 +18,6 @@
 User=root
 KillMode=process
 NotifyAccess=main
-SysVStartPriority=99
 EnvironmentFile=-@sysconfdir@/sysconfig/heartbeat
 EnvironmentFile=-@sysconfdir@/sysconfig/pacemaker
 
